test/runtime/gbt/comm/bigBulkTran.chpl

```diff
@@ -0,0 +1,41 @@
+use Time;
+
+type elemType = uint;
```

Comment:
super minor and it shouldn't matter, but I would prefer to use `int` since I imagine way more user arrays will be ints vs uints.

Reply:
I mainly just wanted the default elemType to be big enough to hold the full initializer values in order to increase the odds of catching mis-copies.  So `int` is fine with me.